src/main/java/com/google/devtools/build/lib/analysis/SourceManifestAction.java

```diff
@@ -118,28 +118,32 @@ void writeEntry(
   @VisibleForTesting
   SourceManifestAction(
       ManifestWriter manifestWriter, ActionOwner owner, Artifact primaryOutput, Runfiles runfiles) {
-    this(manifestWriter, owner, primaryOutput, runfiles, /*remotableSourceManifestActions=*/ false);
+    this(manifestWriter, owner, primaryOutput, runfiles, /*remotableSourceManifestActions=*/ null,
```

Comment:
nit: this comment now has the wrong parameter name (maybe just remove it?)

Reply:
Removed it.